utils.groovy

```diff
@@ -82,4 +82,36 @@ def bump_builds_json(stream, buildid, arch, s3_stream_dir) {
     }
 }
 
+// Run in a podman remote context on a builder of the given
+// architecture.
+//
+// Available parameters:
+//    arch:  string -- The architecture of the desired host
+def withPodmanRemoteArchBuilder(params = [:], Closure body) {
+    arch = params['arch']
+    withPodmanRemote(remoteHost: "fcos-${arch}-builder-host-string",
+                     remoteUid:  "fcos-${arch}-builder-uid-string",
+                     sshKey:     "fcos-${arch}-builder-sshkey-key") {
+        body()
+    }
+}
+
+// Run in a cosa remote session context on a builder of the given
+// architecture.
+//
+// Available parameters:
+// session:  string -- The session ID of the already created session
+//    arch:  string -- The architecture of the desired host
+def withExistingCOSARemoteSession(params = [:], Closure body) {
```

Comment:
Bikeshed: "existing" seems wordily redundant here with "with". Also maybe `s/COSA/Cosa/`?

Reply:
I'm toying with the idea of adding a `withCOSARemoteSession()` in a followup that will create and destroy the remote session with a `body()` in between. This function will still have it's uses though so it might be good to keep the `Existing` here in `withExistingCOSARemoteSession()`.

I'm +1 to make it `Cosa` instead of `COSA`.